### ISSUE TITLE
CSS Level 3: False Positive warning for incompatible attribute 

### ISSUE THREAD

**Issue by AdrianSorop** [w3c/css-validator#211]
I have this CSS selector:

```
*[class^='abc'][class~='def'] {
}
```

and I get this validation warning:

`“[class~="def"]”` and `“[class^="abc"]”` are incompatible

The warning should not be given because the @Class attribute value on an element can have multiple space separted values like:

`<span class="abc def"/>`

**Comment by ylafon** [comment]
Indeed, it seems that the original code was made under the assumption that there was only one attribute. Will fix.

**Comment by ylafon** [comment]
Fixed with 2479909f15d1a70d31bbe4cca939383b779e7f4a
